src/platform_impl/linux/drm/mod.rs

```diff
@@ -0,0 +1,107 @@
+use crate::dpi::{PhysicalPosition, PhysicalSize};
+
+pub mod event_loop;
+pub mod window;
+pub use drm::SystemError;
+pub use event_loop::EventLoop;
+pub use event_loop::EventLoopProxy;
+pub use event_loop::EventLoopWindowTarget;
+pub use window::Window;
+
+#[derive(Debug, Copy, Clone, PartialEq, Eq, PartialOrd, Ord, Hash)]
+pub struct DeviceId;
+
+impl DeviceId {
+    pub const unsafe fn dummy() -> Self {
+        DeviceId
+    }
+}
+
+#[derive(Debug, Clone, PartialEq, Eq)]
+pub struct MonitorHandle(drm::control::connector::Info);
+
+impl PartialOrd for MonitorHandle {
+    fn partial_cmp(&self, other: &Self) -> Option<std::cmp::Ordering> {
+        Some(self.0.interface_id().cmp(&other.0.interface_id()))
+    }
+}
+
+impl Ord for MonitorHandle {
+    fn cmp(&self, other: &Self) -> std::cmp::Ordering {
+        self.0.interface_id().cmp(&other.0.interface_id())
+    }
+}
+
+impl MonitorHandle {
+    #[inline]
+    pub fn name(&self) -> Option<String> {
+        Some(String::from("card0"))
```

Comment:
I'm not sure from where compositors get display names, but it'd be nice to have those.

I'd probably returned `None`, since it's not like a name right now, and arbitrary, since user could pick `card1`...

Reply:
Fixed